lib/nimbase.h

```diff
@@ -108,6 +108,8 @@ __clang__
        defined __SUNPRO_C || \
        defined __xlC__
 #  define NIM_THREADVAR __thread
+#elif defined __TINYC__
+#  defined NIM_THREADVAR
```

Comment:
Oops, of cause ``#  define NIM_THREADVAR`` :-(  Can you commit the fix directly in nim-lang repo, or should I make a PR?